servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -5530,6 +5166,7 @@ public JWSDomain getJWSDomain(ResourceContext ctx, String domainName) {
         // policy, service, etc name)
 
         domainName = domainName.toLowerCase();
+        setRequestDomain(ctx, domainName);
```

Comment:
Originally this method doesn't send the domain but I think it is correct to put it (if it exists)